core/src/main/java/org/dcache/nfs/vfs/PseudoFs.java

```diff
@@ -65,17 +69,23 @@
     private final VirtualFileSystem _inner;
     private final ExportFile _exportFile;
     private final RpcAuth _auth;
+    private final sattr3 _attr;
 
     private final static int ACCESS4_MASK =
             ACCESS4_DELETE | ACCESS4_EXECUTE | ACCESS4_EXTEND
             | ACCESS4_LOOKUP | ACCESS4_MODIFY | ACCESS4_READ;
 
     public PseudoFs(VirtualFileSystem inner, RpcCall call, ExportFile exportFile) {
+        this(inner, call, exportFile, null);
+    }
+
+    public PseudoFs(VirtualFileSystem inner, RpcCall call, ExportFile exportFile, sattr3 attr) {
```

Comment:
you are introducing an extra dependency on nfs v3 code in generic layer. I see two solutions - simple/pragmatic and complicated one. Simple will be update nfsv3 code to issue setattr after create. The complicated one will be to update vfs interface to accept Stat with initial arguments in create (and co) where we currently supply only mode. What do you think?  BTW, what client do you use that sends file owner on create? I haven't seen that!